var/spack/repos/builtin/packages/freetype/package.py

```diff
@@ -47,14 +47,13 @@ def headers(self):
         headers.directories = [self.prefix.include.freetype2]
         return headers
 
-    def configure_args(self):
-        args = [
-            "--with-brotli=no",
-            "--with-bzip2=yes",
-            "--with-harfbuzz=no",
-            "--with-png=yes",
-            "--with-zlib=no",
+
+class CMakeBuilder(CMakeBuilder):
+    def cmake_args(self):
+        return [
+            self.define("FT_DISABLE_ZLIB", True),
+            self.define("FT_DISABLE_BROTLI", True),
+            self.define("FT_DISABLE_HARFBUZZ", True),
+            self.define("FT_DISABLE_PNG", True),
+            self.define("FT_WITH_BZIP2", True),
```

Comment:
Are you sure this is correct?
```
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    FT_WITH_BZIP2
```

Reply:
I took it from freetype's cmake file, let me double check.